README.md

```diff
@@ -52,7 +50,11 @@ if you have general questions or bug reports on GeoJS.
 
 Get Involved
 ============
-GeoJS is developed to meet needs of developers and domain-specific researchers,
-and we are always open for collaboration. We have been doing open source development
-for the last 17 year and understand and appreciate what it takes to develop open
-source software. So fork GeoJS, do great things and let us know if we can help.
+We are always open for collaboration. Kitware has been doing open source development
+for the last 20 year and understand and appreciate what it takes to develop open
```

Comment:
I think this reads better if `understand` and `appreciate` are `understands` and `appreciates`.

Reply:
makes sense, I changed it accordingly. Have a look and let me know if anything else we want to address.